docs/EN_US/ContainerizedHPCC/ContainerizedMods/AzureDeployChapter.xml

```diff
@@ -0,0 +1,645 @@
+
+      <sect3 id="AZ_INstalling-the-HPCCComponents">
+        <title>Installing the HPCC Systems components</title>
+
+        <para>In order for a even a basic installation to succeed, it must
+        have some type of storage enabled. There is more information on
+        storage in the <emphasis>Containerized HPCC Systems</emphasis>
+        documentation. The following steps will create ephemeral storage using
+        the azstorage utility that will allow the HPCC Systems to start and
+        run but will not persist. Please see the section <emphasis>Persistent
+        Storage</emphasis> in the <emphasis>Containerized HPCC
+        Systems</emphasis> documentation for more information.</para>
+
+        <para>To Install the hpcc-azurefile chart:</para>
+
+        <programlisting>helm install azstorage hpcc/hpcc-azurefile</programlisting>
+
+        <para>The goal here is to get the default values from this
+        <emphasis>azstorage</emphasis> chart and create a customization file
+        that will pass in the appropriate values to the HPCC Systems
+        instance.</para>
+
+        <para>Copy the output from the helm install command that you issued in
+        the previous step, from the <emphasis role="bold">storage:</emphasis>
+        parameter through the end of the file and save the file as
+        <emphasis>mystorage.yaml</emphasis>. The
+        <emphasis>mystorage.yaml</emphasis> file should look very similar to
+        the following:</para>
+
+        <programlisting># Pass a values yaml to hpcc helm charts to use these existing claims.
+# e.g.:
+
+storage:
+  planes:
+  - name: dali
+    pvc: dali-azstorage-hpcc-azurefile-pvc
+    prefix: "/var/lib/HPCCSystems/dalistorage"
+    category: dali
+  - name: dll
+    pvc: dll-azstorage-hpcc-azurefile-pvc
+    prefix: "/var/lib/HPCCSystems/queries"
+    category: dll
+  - name: sasha
+    pvc: sasha-azstorage-hpcc-azurefile-pvc
+    prefix: "/var/lib/HPCCSystems/sasha"
+    category: sasha
+  - name: data
+    pvc: data-azstorage-hpcc-azurefile-pvc
+    prefix: "/var/lib/HPCCSystems/hpcc-data"
+    category: data
+  - name: mydropzone
+    pvc: mydropzone-azstorage-hpcc-azurefile-pvc
+    prefix: "/var/lib/HPCCSystems/dropzone"
+    category: lz
+
+
+sasha:
+  wu-archiver:
+    plane: sasha
+  dfuwu-archiver:
+    plane: sasha
+</programlisting>
+
+        <para>We can now use this <emphasis>mystorage.yaml</emphasis> file to
+        pass in these values when we start up our HPCC Systems cluster.</para>
+      </sect3>
+
+      <sect3>
+        <title>Enable Access the ESP Services</title>
+
+        <para>To access your HPCC Systems cloud instance you must enable the
+        visibility of the ESP services. As delivered the ESP services are
+        private with only local visibility. In order to enable global
+        visibility we will be installing the HPCC Systems cluster using a
+        customization file to override those values. There is more information
```

Comment:
In order to enable global visibility, we will be installing the HPCC Systems cluster using a customization file to override the ESP dictionary.